tests/console/hostname.pm

```diff
@@ -22,6 +22,8 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    script_run "systemctl restart wicked.service";
+    sleep 10;
```

Comment:
hm, both of these lines really should not be here, what was your intention? Calling this in the test "hostname" is very confusing.